tests/rptest/services/redpanda.py

```diff
@@ -186,6 +186,9 @@
     re.compile("finject - .* flush called concurrently with other operations")
 ]
 
+# Largest allocation allowed in during a test
```

Comment:
nit: the largest allocation allowed is 1 less than this value

Reply:
Nice find, making it the actual largest alloc allowable.